ballista/rust/core/src/execution_plans/shuffle_writer.rs

```diff
@@ -432,30 +433,31 @@ fn result_schema() -> SchemaRef {
     ]))
 }
 
-struct ShuffleWriter<'a> {
+struct ShuffleWriter {
     path: String,
-    writer: FileWriter<'a, File>,
+    writer: FileWriter<BufWriter<File>>,
     num_batches: u64,
     num_rows: u64,
     num_bytes: u64,
 }
 
-impl<'a> ShuffleWriter<'a> {
+impl ShuffleWriter {
```

Comment:
Please merge arrow2 master for the change of https://github.com/jorgecarleitao/arrow2/commit/4c2f4dca0097f702d9c099ac0e4eb1b8cfefcd18